airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -1783,6 +1783,10 @@ class PropertyHistory(Stream):
     scopes = {"crm.objects.contacts.read"}
     properties_scopes = {"crm.schemas.contacts.read"}
 
+    @property
+    def cursor_field_datetime_format(self):
+        return "x"
```

Comment:
can you give an example of the cursor value?

Reply:
Cursor value is a timestamp in milliseconds (`1700681340514`) and is supported in this function: [`filter_by_state`](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py#L837)

Reply:
Can you add the part of comment about timestamp from doc string for this method also.

Reply:
Done